### PR TITLE
Extend operand permissions

### PR DESCRIPTION
**Pull request by Pejdzor**

### CODE REVIEW

File: config/rbac/role_operands.yaml [hunk 1]
```diff
@@ -16,6 +16,14 @@ rules:
       - "apps"
     resources:
       - deployments
+      - replicasets
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - "batch"
```
Comment (Jan-Olszowka) [inline config/rbac/role_operands.yaml:24]: Are quotes needed here? In above permission files you didn't add them.

Comment (Jan-Olszowka) [inline config/rbac/role_operands.yaml:24]: In addition - why do we need batch apiGroup?

Comment (Pejdzor) [reply config/rbac/role_operands.yaml:24]: Okay, so in this file I follower convention - all other apiGroups were in double quotes, so did I.
The other file is generated automatically when you do `make manifests`.

Comment (Pejdzor) [reply config/rbac/role_operands.yaml:24]: Jobs and CronJobs are part of this apiGroup. To collect data about Jobs and CronJobs, LS need access to batchApi

**Review by Jan-Olszowka — APPROVED** [review]

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -635,9 +644,17 @@ spec:
...
               verbs:
                 - get
                 - list
+            - apiGroups:
+                - batch
+              resources:
+                - jobs
+              verbs:
+                - list
+                - get
```
Comment (alicenoknow) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:657]: q -> do we need both role and cluster role? shouldn't cluster role be enough?

**Review by alicenoknow — APPROVED** [review]
LG

**Pull request merged**